src/sentry/incidents/logic.py

```diff
@@ -461,3 +464,95 @@ def get_incident_suspect_commits(incident):
                     continue
                 seen.add(commit.id)
                 yield commit.id
+
+
+DEFAULT_ALERT_RULE_RESOLUTION = 1
```

Comment:
What will this do in the longer term?

Reply:
This is basically the frequency of updates on the snuba size, in other words the bucket size.

At the moment, we're not actually allowing the user to change the window size in the UI, although that might change. If we allow a larger window, then we'll want to auto calculate the resolution based on it (like maybe we don't want to have to store 1440 minute sized buckets for a day long window, for example).

Reply:
Ok. I can see customers wanting other window sizes in the future. I know datadog/newrelic support windows like 5,10,15,30 minutes as a way to prevent flappy alerts.